src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,193 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Wrapper for the Tasks.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+    private Storage storage;
+    private Ui ui;
+
+    public TaskList(Storage storage, Ui ui) {
+        this.tasks = new ArrayList<>(100);
+        this.storage = storage;
+        this.ui = ui;
+
+        this.storage.initialiseTasks(tasks);
+    }
+
+    public int sizeOf() {
```

Comment:
Might be good to give this a better name
```suggestion
    public int getTasklistSize() {
```